.github/CODEOWNERS

```
@@ -1,3 +1,3 @@
 * @bcumming @msimberg @RMeli
-docs/software/sciapps/cp2k @abussy @RMeli
-docs/software/communication @msimberg
+docs/software/sciapps/cp2k.md @abussy @RMeli
+docs/software/communication.md @msimberg
```

```suggestion
docs/software/sciapps/cp2k.md @abussy @RMeli
docs/software/communication @msimberg
```
Good catch on CP2K, thank you! I think the latter should stay as it is though? It's a directory.